### ISSUE TITLE
Взаимодействие с GitHub

### ISSUE THREAD

**Issue by neSpecc** [codex-team/codex#66]
* Вывести на главной странице сайта время последнего коммита в мастер и колличество коммитов
* В профиле пользователя вывести количество коммитов в мастера и дату последнего коммита
* В адмике в списке пользователей вывести количество дней с момента последнего коммита в любую ветку.

**Comment by n0str** [comment]
Так, в общем, если никто эту задачу себе не берет - я ее запиливаю тогда

**Comment by n0str** [comment]
Добавить статистику по каждому проекту Codex
